### PR TITLE
GitHub Actions (CI) for RSpec

### PR DESCRIPTION
**Pull request by georgiybykov**
- GitHub Actions for RSpec

### CODE REVIEW

File: lib/strong_interface/parameters_validator.rb [hunk 1]
```diff
@@ -33,8 +33,10 @@ def valid?
       end
     end
 
-    def description(metod)
-      metod.inspect.scan(/[.|#](\S+?\(.*?\))/).last.first
+    private
+
+    def description(method)
+      method.inspect.scan(/[.|#](\S+?\(.*?\))/).last.first
```
Comment (programyan) [inline lib/strong_interface/parameters_validator.rb]: I prefer not to use the name of one of the internal methods as a name of a local variable :)

Could you please revert this change, or suggest another name for this variable?

File: .ruby-version [hunk 2]
```diff
@@ -0,0 +1 @@
+2.7.1
```
Comment (programyan) [inline .ruby-version]: I don't think so. 2.7.1 is a minimum version. Because I used filter_map.

File: .github/workflows/rspec.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: .ruby-version
```
Comment (programyan) [inline .github/workflows/rspec.yml]: hmm... It needs to use several versions of ruby from 2.7.1 to current. Is it possible on Github Actions?

File: lib/strong_interface/parameters_validator.rb (hunk 1 above)
Comment (georgiybykov) [reply lib/strong_interface/parameters_validator.rb]: Wow, I thought that was a typo.

I use something like `mthd` or `meth` for such cases, however I have reverted that change. This is ok :ok_hand:

File: .ruby-version (hunk 2 above)
Comment (georgiybykov) [reply .ruby-version]: I understand. Just added it due to GHA config but removed it now to use multiple versions of Ruby in CI.

File: .github/workflows/rspec.yml (hunk 3 above)
Comment (georgiybykov) [reply .github/workflows/rspec.yml]: Sure. I have pushed changes to the code :+1:
